dashboard/src/App.vue

```diff
@@ -90,6 +64,26 @@ export default {
 	font-family: "Avenir", Helvetica, Arial, sans-serif;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+	box-sizing: border-box;
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	justify-content: center;
+	margin: 0;
+	padding: 0;
+	height: 100vh;
+}
+
+.Header, .Footer {
+	width: 100%;
+}
+
+main {
+	flex: auto;
+}
+
+.Footer {
+	margin-bottom: 5px;
```

Comment:
The other parts of the web frontend codebase is currently sticking to `em` units. If there a reason in this specific case to use px, let's document it.